gallery/src/components/NxNumberInput/NxNumberInputPage.tsx

```diff
@@ -0,0 +1,113 @@
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTable, NxCode, NxP, NxTextLink, NxH3 } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxNumberInputSimpleExample from './NxNumberInputSimpleExample';
+import NxNumberInputComplexExample from './NxNumberInputComplexExample';
+import NxNumberInputValidationExample from './NxNumberInputValidationExample';
+import NxNumberInputDisabledExample from './NxNumberInputDisabledExample';
+
+const simpleSourceCode = require('./NxNumberInputSimpleExample?raw');
+const complexSourceCode = require('./NxNumberInputComplexExample?raw');
+const validationSourceCode = require('./NxNumberInputValidationExample?raw');
+const disabledSourceCode = require('./NxNumberInputDisabledExample?raw');
+
+const NxNumberInputPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        <NxCode>NxNumberInput</NxCode> is a variation of <NxCode>NxTextInput</NxCode> whose
+        {' '}<NxCode>type</NxCode> attribute is always set to <NxCode>number</NxCode>.
+      </NxP>
+      <NxP>
+        Note: Safari and Firefox do not prevent the user from inputting non-numerical characters.
+      </NxP>
+      <NxH3>Props</NxH3>
+      <NxP>
+        Aside from the hard-coded <NxCode>type</NxCode> attribute <NxCode>NxNumberInput</NxCode> shares all
+        other <NxCode>NxTextInput</NxCode> props.
+      </NxP>
+      <h3>State Helpers</h3>
+      <NxP>
+        <NxCode>NxNumberInput</NxCode> uses <NxCode>nxTextInputStateHelpers</NxCode>.
+      </NxP>
+      <NxH3>Special attributes</NxH3>
+      <NxP>
+        A number input has three special HTML attibutes which are described briefly below and{' '}
+        <NxTextLink external href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number">
+          in detail on MDN.
+        </NxTextLink>
+      </NxP>
+      <NxTable>
+        <NxTable.Head>
+          <NxTable.Row>
+            <NxTable.Cell>Attribute</NxTable.Cell>
+            <NxTable.Cell>Description</NxTable.Cell>
+          </NxTable.Row>
+        </NxTable.Head>
+        <NxTable.Body>
+          <NxTable.Row>
+            <NxTable.Cell>step</NxTable.Cell>
+            <NxTable.Cell>
+              Use the <NxCode>step</NxCode> attribute to change the incrementation value of the up and down arrows of
+              the input.
+            </NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell>min</NxTable.Cell>
+            <NxTable.Cell>
+              The <NxCode>min</NxCode> attribute defines a minimum value for the input. Note that the component does
+              not provide validation of this value, that's up to the consuming project.
```

Comment:
I think the descriptions for `min` and `max` should be a little clearer that they affect the up and down arrows but not manual character entry.  The current description kind of sounds like they do nothing at all

Reply:
> I think the descriptions for `min` and `max` should be a little clearer that they affect the up and down arrows but not manual character entry. The current description kind of sounds like they do nothing at all

Updated here: https://github.com/sonatype/sonatype-react-shared-components/pull/486/commits/9edb21a0f22a49a860e3c929fdf20ab3d5141376 and further here: https://github.com/sonatype/sonatype-react-shared-components/pull/486/commits/6f37aa60edfc3c8ac89e8370fdd562c50ea3e03b